File: Orenes/Models/Pedido.cs
using Newtonsoft.Json;

namespace Orenes.Models
{
    public enum EstadoPedido
    {
        Pendiente,
        EnProceso,
        Entregado
    }
    public class Pedido
    {
        public ICollection<Ubicacion> Ubicaciones { get; set; }

        public int PedidoId { get; set; }
        public int ClienteId { get; set; }
        public string DireccionEntrega { get; set; }

        public EstadoPedido status { get; set; }
        
        
        [JsonIgnore]
        public Cliente Cliente { get; set; }

    }
}

File: PRUEBATECNICA/angular-Pruebatecnica/src/app/API/models/pedido.ts
/* tslint:disable */
/* eslint-disable */
import { Cliente } from './cliente';
import { EstadoPedido } from './estado-pedido';
import { Ubicacion } from './ubicacion';
export interface Pedido {
  cliente?: Cliente;
  clienteId?: number;
  direccionEntrega?: null | string;
  pedidoId?: number;
  vehiculoId?: number;
  status?: EstadoPedido;
  ubicaciones?: null | Array<Ubicacion>;
}


